src/transformers/models/videomae/feature_extraction_videomae.py

```diff
@@ -97,6 +97,9 @@ def normalize_video(self, video, mean, std):
 
         return (video - mean[None, :, None, None]) / std[None, :, None, None]
 
+    def to_numpy_array_video(self, video, rescale=None, channel_first=True):
+        return [self.to_numpy_array(frame, rescale, channel_first) for frame in video]
+
     def __call__(
         self, videos: ImageInput, return_tensors: Optional[Union[str, TensorType]] = None, **kwargs
```

Comment:
Sorry a bit unrelated to the PR, but why do we have `**kwargs` here?

Reply:
Good question! I'm honestly not sure. cc @NielsRogge

Reply:
+1, seems like `**kwargs` is just leftover code.